src/sentry/services/hybrid_cloud/organization/model.py

```diff
@@ -190,6 +204,16 @@ class RpcOrganization(RpcOrganizationSummary):
 
     default_role: str = ""
 
+    def get_audit_log_data(self):
```

Comment:
Adding this for easy parity with the ORM `Organization` model. Kind of unfortunate if it supports only one call site (in `restore_organization.py`), but it's probably not the last one.